src/main/java/racingcar/util/RaceManager.java

```diff
@@ -0,0 +1,20 @@
+package racingcar.util;
+
+import racingcar.model.Car;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RaceManager {
+    public void movingCars(List<Car> cars) {
+        for(var c : cars) c.move();
+    }
+
+    public List<String> findWinners(List<Car> cars) {
+        int maxLocation = cars.stream().mapToInt(Car::getLocation).max().orElse(0);
```

Comment:
메소드 체이닝은 주로 두 번째부터 줄바꿈을 사용해주고 있어요.
아래 예시 참고 해주세요

```suggestion
        int maxLocation = cars.stream()
            .mapToInt(Car::getLocation)
            .max()
            .orElse(0);
```

Reply:
앗 참고하겠습니다..